app/lib/controlled_work_mappings/cw1.yml

```diff
@@ -159,14 +159,16 @@
 - name: Other property 1_2
   type: text
   source: from_attribute
-  attribute: additional_non_smod_properties_value
+  attribute: additional_properties_value
+  form_type: cw1
```

Comment:
This feels redundant. We know the form type is cw1 _because this file is called cw1.yml_